src-docs/src/views/card/card_example.js

```diff
@@ -367,5 +371,32 @@ export const CardExample = {
   </EuiText>
 </EuiCard>`,
     },
+    {
+      title: 'Plain cards',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: cardDisplaySource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: cardDisplayHtml,
+        },
+      ],
+      text: (
+        <Fragment>
+          <p>
+            If you need a card with no borders or shadows pass{' '}
+            <EuiCode language="ts">{'display="plain"'}</EuiCode>.
+          </p>
+        </Fragment>
+      ),
+      props: { EuiCard },
+      demo: <CardDisplay />,
+      snippet: `<EuiCard
+  title="title"
+  display="plain"
+  description="description" />`,
```

Comment:
Nit, I like to group props between consumer content and string props so that places where consumers need to replace the strings with their own content isn't separate by the simple enum prop.

```suggestion
  title="title"
  description="description" 
  display="plain"
/>`,
```